doc-content/enterprise-only/processes/process-instance-migration-con.adoc

```diff
@@ -3,4 +3,4 @@
 
 Process instance migration (PIM) is a standalone service containing a user interface and a back-end. It is packaged as a Quarkus mutable JAR file. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. The user can then apply the migration plan to the running process instance in a specific {KIE_SERVER}.
 
-For more information about the PIM service, see https://github.com/kiegroup/droolsjbpm-integration/tree/master/process-migration-service[Process Instance Migration Service] in https://github.com/kiegroup[_KIE (Drools, OptaPlanner and jBPM )_].
+For more information about the PIM service, see https://github.com/kiegroup/process-migration-service[Process Instance Migration Service] in https://github.com/kiegroup[_KIE (Drools, OptaPlanner and jBPM )_].
```

Comment:
I am not sure referring to the entire source repo as "for more information about PIM" is a good idea

Reply:
As this page will show the main README I don't think it's a bad idea.

Reply:
Maybe add "see the README file in..."?

Reply:
Thank you both for the comments, I will add "see the README file in".